src/nyc_trees/apps/event/templates/event/event_list_page.html

```diff
@@ -10,7 +10,10 @@
   <div class="map-sidebar">
     <h2>Events</h2>
     <p class="pageheading-description-detail">
+      NYC Parks and TreesCount! partner groups hosted more than 900 events in 2015.
+<!--
       Check out our upcoming TreesCount! events and sign up to map street trees with fellow volun<strong>treers</strong>!
+-->
     </p>
     <p class="pageheading-description-detail">Click to view as a list or find an event nearby using the map.</p>
```

Comment:
I think we should comment this text out as well, since both the map and the list will be empty over the winter until Parks starts loading events for the spring.